services/web/client/source/class/qxapp/component/filter/TagsFilter.js

```diff
@@ -110,6 +91,24 @@ qx.Class.define("qxapp.component.filter.TagsFilter", {
       delete this.__tagButtons[tagName];
       // Dispatch
       this._filterChange(this.__activeTags);
+    },
+
+    _addOption: function(tagName) {
```

Comment:
Same here. We use one underscore for protected and two for private

Reply:
This one is used in child class StudyFilter